test/interface/repgradelbo.jl

```diff
@@ -9,7 +18,24 @@ using Test
 
     (; model, μ_true, L_true, n_dims, is_meanfield) = modelstats
 
-    q0 = TuringDiagMvNormal(zeros(Float64, n_dims), ones(Float64, n_dims))
+    q0 = MeanFieldGaussian(zeros(n_dims), Diagonal(ones(n_dims)))
+
+    @testset "basic" begin
+        @testset for adtype in AD_repgradelbo_interface, n_montecarlo in [1, 10]
+            obj = RepGradELBO(n_montecarlo)
+            _, _, stats, _ = optimize(
+                rng,
+                model,
+                obj,
+                q0,
+                10;
+                optimizer=Descent(1e-5),
+                show_progress=false,
+                adtype=adtype,
+            )
+            @assert isfinite(last(stats).elbo)
```

Comment:
Do we have a test somewhere that tests the correctness of the result more robustly? Like a case where we can compare to the exact analytical result, for instance.

Reply:
The correctness is handled by the tests under `/test/inference`. Here, the intent is just to check that the interface isn't completely broken.